spec/virtual_total_spec.rb

```diff
@@ -35,6 +53,14 @@
           expect(author_query.map(&:total_books)).to match_array([3, 1, 2])
         end.to match_query_limit_of(1)
       end
+
+      it "with no associated records calculates totals with attribute" do
+        Author.create
+        expect do
+          query = Author.select(:id, :total_books).load
+          expect(query.map(&:total_books)).to match_array([0])
+        end.to match_query_limit_of(1)
```

Comment:
Took me a minute to grok this test, but I like that it is added.  👍

Reply:
@NickLaMuro do you want the load outside of the `expect`?

Reply:
This was mostly a "👍 comment" for what it is worth.  Most of my confusion came from where `[0]` was coming from (instead of `[]`), but then I realized it was part of `Author.create` above...

Didn't need to change, but looks like you already did.